packages/story-editor/src/components/audioPlayer/index.js

```diff
@@ -96,8 +97,21 @@ function AudioPlayer({ title, src, mimeType }) {
   return (
     <Wrapper>
       {/* eslint-disable-next-line jsx-a11y/media-has-caption, styled-components-a11y/media-has-caption */}
-      <Audio crossOrigin="anonymous" loop ref={playerRef}>
+      <Audio crossOrigin="anonymous" loop ref={playerRef} id={audioId}>
         <source src={src} type={mimeType} />
+        {tracks &&
+          tracks.map(({ srclang, label, track: trackSrc, id: key }, i) => (
```

Comment:
Minor - think srclang should be `srcLang` to keep it consistent with the other vars i.e. trackSrc , playerRef etc...

Reply:
I am going to fix this in an follow up PR.

Reply:
@timarney Created a follow up PR - https://github.com/GoogleForCreators/web-stories-wp/pull/10548